apps/climatemappedafrica/src/payload/blocks/HowItWorks.js

```diff
@@ -0,0 +1,86 @@
+import { slateEditor } from "@payloadcms/richtext-slate";
+
+import image from "../fields/image";
+import linkGroup from "../fields/links/linkGroup";
+import richText from "../fields/richText";
+
+const HowItWorks = {
+  slug: "how-it-works",
+  imageURL: "/images/cms/blocks/how-it-works.png",
+  fields: [
+    {
+      name: "title",
+      type: "text",
+      required: true,
+      localized: true,
+    },
+    richText({
+      name: "description",
+      label: {
+        en: "Description",
+      },
+      editor: slateEditor({
+        admin: {
+          elements: ["link"],
+          leaves: ["bold", "code", "italic", "underline"],
+        },
+      }),
+      required: true,
+      localized: true,
+    }),
+    {
+      name: "video",
+      type: "group",
+      fields: [
+        {
+          name: "url",
+          type: "text",
+          label: {
+            en: "Youtube Video URL",
+          },
+          required: true,
+          localized: true,
+        },
+        {
+          name: "type",
+          type: "text",
+          required: true,
+          defaultValue: "video/youtube",
+          admin: {
+            description: "The type of video. e.g. video/mp4, video/youtube",
+            hidden: true,
+          },
+        },
```

Comment:
If this doesn't change, why are we even asking it (or why not just [hide](https://payloadcms.com/docs/fields/overview#admin-config) it if you want still want the value stored in the CMS)/